pom.xml

```diff
@@ -49,6 +49,96 @@
 			<version>2.4.4</version>
 		</dependency>
 
-	</dependencies>
+	 <!-- Dependencies for Git Module <added by Sujeet, starting from here>  -->
+    
+    <dependency>
```

Comment:
You should probably, as a team, come up with an indentation standard (I think four spaces is typical for pom.xml).